Package.swift

```diff
@@ -1355,5 +1346,5 @@ func googleAppMeasurementDependency() -> Package.Dependency {
     return .package(url: appMeasurementURL, branch: "main")
   }
 
-  return .package(url: appMeasurementURL, exact: "10.10.0")
+  return .package(url: appMeasurementURL, branch: "main")
```

Comment:
Not this?

Reply:
correct. not sure how that I did that.

Reply:
oh, i ran `scripts/setup_spm_test.sh` at one point. good catch!